src/tyr/route_serializer_valhalla.cc

```diff
@@ -122,8 +125,9 @@ json::MapPtr summary(const valhalla::Api& api) {
   }
 
   auto route_summary = json::map({});
-  route_summary->emplace("time", json::fp_t{time, 3});
-  route_summary->emplace("length", json::fp_t{length, 3});
+  route_summary->emplace("time", json::fp_t{route_time, 3});
+  route_summary->emplace("length", json::fp_t{route_length, 3});
+  route_summary->emplace("cost", json::fp_t{route_cost, 3});
```

Comment:
Added `cost` to the summary part of the response. This was helpful to add in this PR to compare the costs of routes that changed in RAD tesing.

Reply:
we should do this for the leg summary as well on line 327

Reply:
Done